src/static/css/module/loan-comparison.less

```diff
@@ -751,11 +747,66 @@
       &.label-cell {
         padding-left: @grid_gutter-width;
       }
+      
+      &.link {
+        span {
+          display: none;
+        }
+      }
     }
   
-    tbody tr {
+    tr {
       border: none;
+      
+      &.padded-row {
+        td.label-cell {
```

Comment:
after testing we should revisit the nesting in this section and see if we can reduce it.